### DISCUSSION TITLE
404 error

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37712] [Other Feature Feedback, Questions, & Ideas]
Hi, i m trying to host my site on GitHub but i'm facing page not found 404 error while navigating to other page please help as soon as possible
- a tags are correct
- repo name is exact same as my username
But im still facing the 404 error

![Screenshot_20221029-170905](https://user-images.githubusercontent.com/88527795/198831479-6112be48-a5ec-4688-9d18-eb930a4d324d.jpg)

**Comment by airtower-luna** [comment]
If you mean [this repository](https://github.com/vipervikxey/vipervikxey.github.io), the site seems to be up now. It probably just took some time.

**Reply by ghost** [reply]
Thank you it's working now
